pkg/executor/remote.go

```diff
@@ -189,7 +187,7 @@ func (remote Remote) Execute(command string) (TaskHandle, error) {
 		stdoutFilePath:   stdoutFile.Name(),
 		stderrFilePath:   stderrFile.Name(),
 		host:             remote.targetHost,
-		exitCode:         exitCode,
+		exitCode:         &exitCode,
```

Comment:
Can we kill the pointer in `remoteTaskHandle`? This will help to reduce _pointer_ bugs.